### ISSUE TITLE
Tiling Sprite Doesn't work when using local files not from a CDN

### ISSUE THREAD

**Issue by Codaea** [hairyf/vue3-pixi#105]
worked for `<sprite/>`, but not this. 

```ts
<template>
    <tiling-sprite
    texture="@/assets/pain.jpeg"
      :anchor="0.5"
      :x="120"
      :y="120"
    />
</template>
```

however, using a CDN it renders fine.

```ts
<template>
    <tiling-sprite
    texture="https://pixijs.com/assets/p2.jpeg"
      :anchor="0.5"
      :x="120"
      :y="120"
    />
</template>
```
here is the log i would get on render

[error.log](https://github.com/hairyf/vue3-pixi/files/14353269/error.log)

this could be me being stupid, or something else. anything's appreciated. Thanks!

**Comment by Codaea** [comment]
Nevermind, It needs to be in the public folder. can't belive it took me 4 hours to figure that out. 

TLDR; you cant get from assets folder but public is fine.

**Comment by Codaea** [comment]
however, debug messages would still be appreciated :)
